crm_lead_probable_revenue/models/crm_lead.py

```diff
@@ -8,11 +8,11 @@
 class CrmLead(models.Model):
     _inherit = 'crm.lead'
 
+    @api.depends('planned_revenue', 'probability')

```

Comment:
Insert @api.depends after @api.one.  It may work either way, but let's keep the style in sync with Odoo standard code.